rclpy/rclpy/subscription_options.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
+
+
+class SubscriptionOptions(_rclpy.rmw_subscription_options_t):
+    pass
```

Comment:
I was expecting that abstraction class such as `QoSProfile` with initialization, setter and getter accessors? so that it would be useful as python client library, and conceal implementation details with C/C++ language.

Reply:
I was trying to keep the implementation more in line with the rclcpp interface, but can look at the changes this might require.